ddp-workspace/projects/ddp-sdk/src/lib/components/cookies/cookiesPreferencesModal/cookiesPreferencesModal.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Inject } from '@angular/core';
```

Comment:
This component is covered by another ticket, so I'm not reviewing this closely, but this also appears not to translate.

Reply:
Done in baa19cc

Added translations of mat-cells with cookies information